homeassistant/components/homewizard/manifest.json

```diff
@@ -11,7 +11,7 @@
   "documentation": "https://www.home-assistant.io/integrations/homewizard",
   "iot_class": "local_polling",
   "loggers": ["homewizard_energy"],
-  "quality_scale": "platinum",
+  "quality_scale": "bronze",
```

Comment:
We can't allow for that, as this integration has devices in the WWHA program.

As an alternative, we could introduce an intermediate working branch.

Reply:
Sure! How would that work here?

Reply:
Good question. So I create a branch, and you target your PRs against it. 

Once all is done, you create a new PR merging that intermediate branch will all changes into dev in a single go :)

Reply:
Sounds simple, great plan 👍

Reply:
Done!